### ISSUE TITLE
Seeking help for PyTorch conversion

### ISSUE THREAD

**Issue by lijingrs** [huggingface/candle#2401]
`
class TextCNNModel(nn.Module):
    def __init__(self):
        super(TextCNNModel, self).__init__()
        self.embedding = nn.Embedding(
            num_embeddings=config.num_embeddings,
            embedding_dim=config.embedding_dim,
            padding_idx=config.padding_idx
        )
        self.convs = nn.ModuleList(
            [nn.Conv2d(1, config.num_filters, (k, config.embedding_dim)) for k in config.filter_sizes]
        )
        self.dropout = nn.Dropout(config.dropout)
        self.fc = nn.Linear(config.num_filters * len(config.filter_sizes), config.target_len)

    def conv_and_pool(self, x, conv):
        x = F.relu(conv(x)).squeeze(3)
        x = F.max_pool1d(x, x.size(2)).squeeze(2)
        return x

    def forward(self, x):
        out = self.embedding(x)
        out = out.unsqueeze(1)  
        out = torch.cat([self.conv_and_pool(out, conv) for conv in self.convs], 1)
        out = self.dropout(out)
        out = self.fc(out)
        return out

    def model_eval(self, x):
        out = self.embedding(x)
        out = out.unsqueeze(1)
        out = torch.cat([self.conv_and_pool(out, conv) for conv in self.convs], 1)
        out = self.fc(out)
        return out

`

I am in the process of converting a Python codebase that is based on PyTorch into Rust using the Candle library. I've encountered some blocking issues and I'm looking for assistance. Can anyone help me with this?
The APIs are incompatible between the two, and some methods are not found.

**Comment by lijingrs** [comment]
import torch.nn.functional as F
def conv_and_pool(self, x, conv):
    x = F.relu(conv(x)).squeeze(3)
    x = F.max_pool1d(x, x.size(2)).squeeze(2)
    return x

**Comment by omkar-12bits** [comment]
join Huggingface discord server and ask for help in candle channel , i will try to help you out there.

**Comment by lijingrs** [comment]
> join Huggingface discord server and ask for help in candle channel , i will try to help you out there.

All right, Thanks!
